### PR TITLE
Add calaveras testnet and remove baikal

### PR DESCRIPTION
**Pull request by matkt**
Signed-off-by: Karim TAAM <karim.t2am@gmail.com>

## PR description

Add calaveras testnet and remove baikal for London

## Changelog

- [x] I thought about the changelog and included a [changelog update if required](https://wiki.hyperledger.org/display/BESU/Changelog).

### CODE REVIEW

**Review by abdelhamidbakhta — APPROVED** [review]
Looks good to me.

File: config/src/main/resources/calaveras.json [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
+    "constantinopleBlock": 0,
     "petersburgBlock": 0,
     "istanbulBlock": 0,
     "berlinBlock": 0,
     "londonBlock": 500,
     "clique": {
-      "blockperiodseconds": 30,
-      "epochlength": 30000
+      "period": 30,
+      "epoch": 30000
```
Comment (garyschulte) [inline config/src/main/resources/calaveras.json]: shouldn't these be epochlengh and blockperiodseconds ?

**Review by garyschulte — APPROVED** [review]
lgtm

File: config/src/main/resources/calaveras.json (hunk 1 above)
Comment (matkt) [reply config/src/main/resources/calaveras.json]: Yes I updated just before your comment 😄

**Pull request merged**